Project.toml

```diff
@@ -0,0 +1,26 @@
+name = "ConicBenchmarkUtilities"
+uuid = "e95a7839-07fb-532d-9a0e-071766bb5168"
+authors = ["Miles Lubin"]
+repo = "https://github.com/JuliaOpt/ConicBenchmarkUtilities.jl.git"
+version = "0.3.2"
+
+[deps]
+GZip = "92fee26a-97fe-5a0c-ad85-20a5f3185b63"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+
+[compat]
```

Comment:
The tag PR isn't automatically mergeable without version bounds on all dependencies, e.g., `GZip`.